assets/css/style.scss

```diff
@@ -34,10 +34,14 @@ main.page-content {
   color: #111;
 }
 
+.table-borderless, .table-borderless *, .table-borderless * * {
+  border-style : hidden !important;
+  background-color: rgba(0,0,0,0) !important;
```

Comment:
Why are you using `!important` here? I've always had the vague impression that it shouldn't be necessary in most cases.

Reply:
The Jekyll CSS loads after this CSS (no way to change that), so `!important` has to be used to override it.